src/tests/validity/len_string.ab

```diff
@@ -0,0 +1,6 @@
+// Output
+// 5
+
+main {
+    echo len "hello"
```

Comment:
This fails to parse for me:
```
$ amber len_string.ab
 ERROR  Identifier 'len' is a reserved keyword
at len_string.ab:5:10

4| main {
5|     echo len "hello"
6| }
```

Reply:
yeah, I don't understand this error as I followed the instructions to add a new builtin.

Reply:
i mean you can clearly see the issue

https://github.com/amber-lang/amber/blob/cd0338af4431c00cae51a64f6650c8a7ac62b10f/src/modules/variable/mod.rs#L81-L84